views/businessForms/documentQuantity.ejs

```diff
@@ -58,24 +58,30 @@
 
                     <label class="form-label-bold" for="documentCount">Number of documents</label>
                     <p>How many documents do you want to legalise?</p>
+                    <span class="form-hint">Each document costs £<%= doc_cost%> <% if (req.session.appType === 2) { %> . Maximum 20 documents per application. <% } %></span>
                     <legend id="doc_count" class="visuallyhidden"> Your reference (optional) </legend>
 
-
                     <span class="error-message">
-                                        <span class="error-message">
-                                            <% if (error_report) {  %>
-                                            <%= error_report[0][0].errMsgs[0].fieldError %>
-                                            <% } %>
-                                        </span>
-                                    </span>
+                        <span class="error-message">
+                            <% if (error_report) {  %>
+                                <%= error_report[0][0].errMsgs[0].fieldError %>
+                            <% } %>
+                        </span>
+                    </span>
+
                     <input type="number" aria-required="true" step="any"
                            <% if (submit_status===true) { %>disabled="disabled"<% } %>
                            id="documentCount" name="documentCount"
                            value="<% if (form_values && !selected_docs_count) { %><%= (form_values.doc_count || '') %><% } else if (!form_values && selected_docs_count) { %><%= (selected_docs_count || '')%><% } %>"
-                           class="number govuk-input govuk-!-width-one-quarter"/>
+                           class="number govuk-input govuk-!-width-one-quarter"
+                            <% if (req.session.appType === 2) { %>
+                                max="20"
```

Comment:
Perhaps just rename this to maxPremDocs then change     
`var maxAttr = $(this).attr('max');    ` 
to
`var maxAttr = $(this).attr('maxPremDocs');`